lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -259,6 +261,145 @@ class ExportCommand < HammerCLIKatello::SingleResourceCommand
       end
     end
 
+    class ExportCommand < HammerCLIForeman::Command
+      include HammerCLI::Messages
+      include HammerCLIKatello::LocalHelper
+
+      command_name "export"
+
+      success_message _("Content view export is available in %{directory}.")
+      failure_message _("Could not export the content view")
+
+      option "--id", "ID", _("Content View Version numeric identifier")
+      option '--export-dir', 'EXPORT_DIR', _("Directory to put content view version export into.")
+
+      validate_options do
+        option(:option_export_dir).required
+      end
+
+      build_options
+
+      class ContentViewIdParamSource
+        def initialize(command)
+          @command = command
+        end
+
+        def get_options(_defined_options, result)
+          if result['option_id'].nil? && result['option_name']
+            result['option_id'] = @command.resolver.content_view_id(result)
+          end
+          result
+        end
+      end
+
+      def option_sources
+        super << ContentViewIdParamSource.new(self)
+      end
+
+      def execute
+        cvv = content_view_version
+        repositories = cvv['repositories'].collect do |repo|
+          repository(repo['id'])
+        end
+
+        check_repo_download_policy(repositories)
+
+        repositories.each do |repo|
+          repo['packages'] = packages(repo['id'])
+          repo['errata'] = errata(repo['id'])
+        end
+
+        json = export_json(cvv, repositories)
+        create_tar(cvv, repositories, json)
+        return 0
```

Comment:
@akofink This seems kind augly, but couldnt find another work around

Reply:
I think you want `return HammerCLI::EX_OK`, and that makes it look better :)